src/Shared/NativeMethodsShared.cs

```diff
@@ -1241,6 +1241,48 @@ internal static int GetParentProcessId(int processId)
             return myChildren;
         }
 
+        internal static string GetModuleFileName(string fileName)
```

Comment:
Noticed this execution path isn't covered at all by tests in ```StripTypeLibNumberFromPath```, what would be a good way to test this?